src/Cms/Plugin.php

```diff
@@ -79,25 +73,24 @@ public function authorsNames(): string
 	}
 
 	/**
-	 * @return array
+	 * Returns the associative array of plugin extensions
```

Comment:
```suggestion
	 * Returns the associative array of extensions the plugin bundles
```

Might make the distinction between plugin vs. extension even clearer